components/chat/header/db-selector.tsx

```diff
@@ -36,7 +37,7 @@ function DBSelector() {
     >
       {dbOpts.map((item) => (
         <Select.Option key={item.value}>
-          <Icon component={DBSvg} className="mr-1" />
```

Comment:
not only show DB icon rather than special database icon, such as `mysql`, `duckdb`